test/backend/routes.queries.spec.js

```diff
@@ -158,12 +158,13 @@ describe('Routes:', () => {
 
             return POST('queries', queryObject)
                 .then(assertResponseStatus(201))
-                .then(getResponseJson).then(json => {
-                    assert.deepEqual(json, {});
-                    return GET('queries');
-                })
-                .then(getResponseJson).then(json => {
-                    assert.deepEqual(json, [queryObject]);
+                .then(getResponseJson).then((json) => {
+                    assert.deepEqual(omit('uids', json), omit('uids', queryObject));
```

Comment:
This is the only line that needed updating in this test.
Let's revert everything else.

---

Re the `uids`, is it possible to correct te `const uids` above?